testdata/crc32_amd64.golden

```diff
@@ -2,85 +2,85 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-#define NOSPLIT 4
-#define RODATA 8
+#define	NOSPLIT 4
```

Comment:
I'm not sure where `#include` and `#define` are handled in the actual formatting code, but it seems strange to have a tab here.  Perhaps leave the separator as a single space here?